training/mochi-1/prepare_dataset.py

```diff
@@ -0,0 +1,680 @@
+#!/usr/bin/env python3
```

Comment:
It looks mostly similar to our existing script, so I think the changes are all correct due to the minimal diff - very cool!